Dependencies/xr/Source/ARKit/XR.mm

```diff
@@ -946,6 +946,8 @@ void UpdateXRView(MTKView* activeXRView) {
 
             // Finalize rendering here & push the command buffer to the GPU.
             [currentCommandBuffer commit];
+			
+            [currentCommandBuffer waitUntilCompleted];
```

Comment:
What validation error does this fix? I don't think this is good for perf. I think we used to do this and tried to get rid of it, because it blocks this thread until the command buffer has been executed by the gpu.

Reply:
I know but I'd like to narrow down the issue to see if it's this command buffer that's causing all the crashes.